k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeUtility.java

```diff
@@ -32,6 +34,8 @@
     private static final String HEADER_PARAM_FORMAT = "format";
     private static final String HEADER_FORMAT_FLOWED = "flowed";
 
+    private static int filledBytes = 0;
```

Comment:
A Utility class should not store temporal information. This needs some more thought.